setup_dev_env.sh

```diff
@@ -36,11 +36,25 @@ fi
 # define a python version to initialise the conda environment.
 # by default we assume python 3.7.
 python_version=${PYTHON_VERSION:-"3.7"}
+pytorch_version=${PYTORCH_VERSION:-"1.6.0"}
+pytorch_mode=${PYTORCH_MODE:-""}  # use `cpuonly` for CPU or leave it in blank for GPU
```

Comment:
Would this auto-detect the device?

Probably something like:
```python
try:
    nvidia-smi
    pytorch+cuda
except:
    pytorch+cpu
```

Reply:
It's better to specify by hand in case we want to test cpuonly